pdctl/command/store_command.go

```diff
@@ -50,6 +51,16 @@ func NewDeleteStoreCommand() *cobra.Command {
 	return d
 }
 
+// NewSetStoreStateCommand returns a state subcommand of storeCmd.
+func NewSetStoreStateCommand() *cobra.Command {
+	d := &cobra.Command{
+		Use:   "state <store_id> [Up|Offline|Tombstone]",
```

Comment:
can we set the tombstone state directly? Seem it is dangerous.

Reply:
We already have API to let user to set a store's state to 'Tombstone' (from any state)... See: https://github.com/pingcap/pd/blob/73fa4c4f2c394817fbb511afb06a5933b845167e/server/cluster.go#L486-L490

Reply:
Got it.

Reply:
But can we transfer a tombstone store to up again?

Reply:
I think we can. The 'tombstone' is just a flag in pd-server that prevent data been transferred to the store, and make the store ignored by any balancers.